packages/client/modules/teamDashboard/components/TeamDashInsights/TopRetroTemplatesCard.tsx

```diff
@@ -36,20 +38,21 @@ const TopRetroTemplatesCard = (props: Props) => {
     <TeamInsightsCard
       teamInsightsRef={insights}
       title='Top Templates'
-      tooltip='The most frequently used retrospective templates on your team'
+      tooltip='The most frequently used retrospective templates on your team for the last 12 months'
     >
       <div className='flex w-full flex-col'>
         {topRetroTemplates.map((template, index) => {
-          const {reflectTemplate} = template
+          const {reflectTemplate, count} = template
           const {name, illustrationUrl} = reflectTemplate
           return (
-            <div
+            <Tooltip
+              text={`Used ${count} times in the last 12 months`}
```

Comment:
+1 you can use the plural utility function here: https://github.com/ParabolInc/parabol/blob/8c68bf771a350b31c7948e6bd11c5c110e30c977/packages/client/utils/plural.ts

Otherwise it says: `Used 1 times` rather than `Used 1 time`